src/lib.rs

```diff
@@ -357,7 +357,7 @@ decl_storage! {
 
         /// Blog by unique blog identificator
        
-        BlogById get(fn blog_by_id): map T::BlogId => Blog<T>;
+        BlogById get(fn blog_by_id): map T::BlogId => Option<Blog<T>>;
```

Comment:
This allows to properly distinguish between default structure right after creation and non existent one, when accessing from the outside of blog module. We get `Some(value)` or `None` , instead of getting default structure implementation. Currently we use this functionality in unit tests to check storage state after extrinsics performed, for example. If we  have a way to access runtime storage specific methods like `::exists()` directly, there is no problem to reverse this change, i think.

Reply:
Yes please reverse this, `exists` does the check fine, and is semantically clearer. Most importantly, having inaccurate type representation decreases safety, e.g. such as inserting `None` value.

Reply:
Fixed here https://github.com/Joystream/substrate-blog-module/pull/1/commits/57e6ed4a65cd61f8551175b6511820702f0191c6